docs/user/spark/pyspark.rst

```diff
@@ -0,0 +1,91 @@
+GeoMesa PySpark
+---------------
+
+GeoMesa provides support for working with data stored in Accumulo using the Spark Python API.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+`Spark`_ 2.0 should be installed.
+
+`Python`_ 2.7 or 3.x should be installed.
+
+`pip`_ or ``pip3`` should be installed.
+
+Installation
+^^^^^^^^^^^^
+
+The ``geomesa_pyspark`` package is not available for download. Build the artifact locally with the profile ``-Ppython``. Then install using ``pip`` or ``pip3`` as below. You will also need to put the geomesa-accumulo-spark-runtime jar in a convenient location. This jar bundles together the client libraries for GeoMesa Spark.
+
+.. code-block:: bash
+
+    mvn clean install -Ppython
+    pip3 install geomesa-spark/geomesa_pyspark/target/geomesa_pyspark-$VERSION.tar.gz
+    cp  geomesa-accumulo/geomesa-accumulo-spark-runtime/target/geomesa-accumulo-spark-runtime_2.11-$VERSION.jar /path/to/
```

Comment:
Incidentally, $VERSION will not be substituted in the built docs; Sphinx substitutions look like: |version|. Unfortunately, the Sphinx substitutions don't work in code blocks or admonitions! That's why you'll sometimes see things like this in the docs: ``in the code below, $VERSION = |version|.``